content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -95,13 +80,25 @@ For example if you followed the instructions to
 [install containerd](/docs/setup/production-environment/container-runtimes/#containerd)
 replace the `containerd-shim-runhcs-v1.exe` is installed at `$Env:ProgramFiles\containerd` with the newly built shim.
 
-We are working to improve this process by enabling HostProcess support directly in 
-[containerd via the CRI api](https://github.com/containerd/containerd/pull/5131) 
-as well as shipping a version of the containerd shim for Windows with support direclty via containerd releases.
+## Limitations
+
+- HostProcess containers require version 1.5.4 or higher of the containerd {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
+- As of v1.22 HostProcess pods can only contain HostProcess containers. This is a current limitation 
+of the Windows OS; non-privileged Windows containers cannot share a vNIC with the host IP namespace.
+- HostProcess containers run as a process on the host and do not have any degree of 
+isolation other than resource constraints imposed on the HostProcess user account. Neither 
+filesystem or Hyper-V isolation are supported for HostProcess containers.
+- Volume mounts are supported and are mounted under the container volume. 
+See [Volume Mounts](./create-hostprocess-pod#volume-mounts)
+- A limited set of host user accounts are available for HostProcess containers by default. 
+See [Choosing a User Account](./create-hostprocess-pod#choosing-a-user-account).
+- Resource limits (disk, memory, cpu count) are supported in the same fashion as processes 
+on the host.
+- Both Named pipe mounts and Unix domain sockets are **not** currently supported and should instead 
+be accessed via their path on the host (e.g. \\\\.\\pipe\\\*)
 
-## Creating a HostProcess Pod
+## Creating a HostProcess Security Policy Config
```

Comment:
```suggestion
## Creating a HostProcess security policy configuration
```

I'm not sure what you create when you follow this step. What is it that the reader creates?

Reply:
Good point, I was thinking of it as 'what to do for HostProcess when creating your configuration'. I'll reword it to 'HostProcess Pod configuration requirements'